src/basic/missing_fs.h

```diff
@@ -57,6 +57,11 @@
 #define CIFS_MAGIC_NUMBER 0xFF534D42
 #endif
 
+/* Not exposed yet. Defined at fs/cifs/smb2glob.h */
+#ifndef SMB2_MAGIC_NUMBER
+#define SMB2_MAGIC_NUMBER 0xFE534D42
```

Comment:
The magic number is equivalent to `CIFS_MAGIC_NUMBER`. So, it is not necessary to add this.

Reply:
will apply the suggested changes.